j2v/generation/generator.py

```diff
@@ -49,7 +49,7 @@ def collect_all_paths(self, current_dict, current_path=None, current_view=None,
         for key, value in current_dict.items():
             if type(key) != str:
                 continue
-            if is_primitive(value):
+            if is_primitive(value) or value is None:
```

Comment:
It'd be good to add one such case to the examples file (and I'd assume, there should be a change in the example output with your addition).

Reply:
I have just added it in commit 8825b46